docs/guides/email/02-basic-email-system.md

```diff
@@ -0,0 +1,755 @@
+tcp      LISTEN    0         100                              192.168.100.6:993             0.0.0.0:*       users:(("dovecot",pid=3808,fd=39))
+tcp      LISTEN    0         100                              192.168.100.6:995             0.0.0.0:*       users:(("dovecot",pid=3808,fd=22))
+tcp      LISTEN    0         100                              192.168.100.6:587             0.0.0.0:*       users:(("master",pid=4066,fd=17))
+tcp      LISTEN    0         100                              192.168.100.6:110             0.0.0.0:*       users:(("dovecot",pid=3808,fd=21))
+tcp      LISTEN    0         100                              192.168.100.6:143             0.0.0.0:*       users:(("dovecot",pid=3808,fd=38))
+tcp      LISTEN    0         100                              192.168.100.6:465             0.0.0.0:*       users:(("master",pid=4066,fd=20))
+tcp      LISTEN    0         128                                       [::]:22                 [::]:*       users:(("sshd",pid=732,fd=4))
+```
+
+Ports occupied by postfix -- 25, 587, 465
+Ports occupied by dovecot -- 993, 995, 110, 143
+
+You can use the `doveadm` command to generate the relevant ciphertext password and insert it into the virtual_users table.
+
+<a id="ap1"></a> 
+
+```bash
+Shell(192.168.100.6) > doveadm pw -s SHA512-crypt -p onetestandone
+{SHA512-CRYPT}$6$dEqUVsCirHzV8kHw$hgC0x0ufah.N0PzUVvhLEMnoww5lo.JBmeLSsRNDkgWVylC55Gk6zA1KWsn.SiIAAIDEqHxtugGZWHl1qMex..
+
+Shell(192.168.100.6) > doveadm pw -s SHA512-crypt -p twotestandtwo
+{SHA512-CRYPT}$6$TF7w672arYUk.fGC$enDafylYnih4q140B2Bu4QfEvLCQAiQBHXpqDpHQPHruil4j4QbLXMvctWHdZ/MpuwvhmBGHTlNufVwc9hG34/
+```
+
+Insert relevant data on the 192.168.100.5 host.
+
+```sql
+Mysql > use mailserver;
+
+Mysql > insert into virtual_users(id,email,password,domain_id) values(1,'frank@mail.rockylinux.me','$6$dEqUVsCirHzV8kHw$hgC0x0ufah.N0PzUVvhLEMnoww5lo.JBmeLSsRNDkgWVylC55Gk6zA1KWsn.SiIAAIDEqHxtugGZWHl1qMex..',1);
+
+Mysql > insert into virtual_users(id,email,password,domain_id) values(2,'leeo@mail.rockylinux.me','$6$TF7w672arYUk.fGC$enDafylYnih4q140B2Bu4QfEvLCQAiQBHXpqDpHQPHruil4j4QbLXMvctWHdZ/MpuwvhmBGHTlNufVwc9hG34/',1);
+```
+
+### Test 
+
+#### User's authentication
+
+Use another Windows10 computer and change its preferred DNS to 192.168.100.7. The author uses foxmail as the mail client here.
+
+On the main screen, select "Other Mailbox" --> "Manual" --> Enter the relevant content to complete. --> "Create"
+
+![test1](./email-images/test1.jpg)
+
+![test2](./email-images/test2.jpg)
+
+#### Send an email
+
+Use this user to attempt to send an email to a leeo user.
+
+![test3](./email-images/test3.jpg)
+
+#### Receive mail
+
+![test4](./email-images/test4.jpg)
+
+### Additional description
+
+* You should have a legitimate domain name (domain)
+* You should apply for an SSL/TLS certificate for your email system
```

Comment:
Other than those few comments and suggestions, this looks really good. I'll give you a chance to edit those and you can let me know when you are ready to merge. Thank you!